package.json

```diff
@@ -47,14 +62,20 @@
     "babel-plugin-module-resolver": "^2.7.1",
     "babel-preset-flow": "^6.23.0",
     "babel-preset-react-native": "1.9.1",
-    "eslint": "^3.15.0",
-    "eslint-config-airbnb": "^14.1.0",
+    "eslint": "^3.19.0",
+    "eslint-config-airbnb": "~15.0.1",
+    "eslint-config-airbnb-base": "~11.2.0",
+    "eslint-import-resolver-babel-module": "^3.0.0",
+    "eslint-plugin-flowtype": "^2.35.0",
     "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^4.0.0",
-    "eslint-plugin-react": "^6.10.0",
-    "eslint-plugin-react-native": "^2.2.1",
+    "eslint-plugin-jsx-a11y": "^5.0.1",
+    "eslint-plugin-react": "^7.1.0",
```

Comment:
@patw0929 did you choose packages "fixed" versions for:
- `eslint-config-airbnb`
- `eslint-plugin-jsx-a11y`
- `eslint-plugin-react`

for any good reason ?

I'm using Atom 1.18.0 and the ESLinter returns an error each time I try to modify a file. After updating these packages versions : https://github.com/Antoine38660/git-point/commit/cfdd16d8786ec8ae7ce32f31a496644b5c0c0060 linter in Atom works again.

I wanted your opinion before submitting a new PR 😄